crates/ironrdp-rdpdr/Cargo.toml

```diff
@@ -1,14 +1,13 @@
 [package]
 name = "ironrdp-rdpdr"
 version = "0.1.0"
-description = "RDPDR static channel for file system implemented as described in MS-RDPEFS"
+description = "RDPDR channel implementation."
```

Comment:
The original description was too verbose?

Reply:
The RDPDR static channel isn't entirely defined in MS-RDPEFS, there are other MS-* which define significant portions of it's functionality such as MS-RDPESC (smartcards) or MS-RDPEPC (printers), so I decided to just keep it high level.

Reply:
Fair enough